root.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$ROOT = '/var/www/html/vinsearch/';
```

Comment:
define('ROOT_PATH', realpath(dirname(__FILE__)));


I would do something like this in case the directory changes

Reply:
Another option is
require_once(__DIR__ . '/../includes/autoload.php');